flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/config/Environment.java

```diff
@@ -69,7 +69,7 @@
 	private DeploymentEntry deployment;
 
 	public Environment() {
-		this.modules = Collections.emptyMap();
+		this.modules = new LinkedHashMap<>();
```

Comment:
i think is better to use Collections.emptyMap() in construct, not change to new LinkedHashMap();

Reply:
we'd better use a linked hashmap to preserve order just in case